src/App.js

```diff
@@ -1,25 +1,63 @@
 import React from 'react';
-
+import { Clock } from './components/Clock';
 import './App.scss';
 
-const App = () => {
-  setInterval(() => {
-    const date = new Date();
+class App extends React.Component {
+  state = {
+    isClockVisible: true,
+    clockName: Math.floor(Math.random() * 1000),
+  }
+
+  getShow() {
+    this.setState({ isClockVisible: true });
+  }
 
+  getHide() {
+    this.setState({ isClockVisible: false });
+  }
+
+  randomName() {
```

Comment:
```suggestion
  getRandomName() {
```
Let's name the methods more meaningfully so their names should display what they actually do